src/response/cache/sorted_set_fetch.rs

```diff
@@ -6,6 +6,8 @@ use momento_protos::cache_client::SortedSetFetchResponse;
 
 use crate::{MomentoError, MomentoResult};
 
+// TODO this needs to be moved to the requests directory
```

Comment:
I made it a separate file because it is used by multiple requests. Are we going to define all responses alongside the requests because they are closely tied to them?

Reply:
yeah i meant to reach out to you about this and get your opinion.

tbh part of my motivation here is just to help us in tracking milestones toward the release - my current thinking is that the `requests` directory will go away entirely. I'd rather not have to surgically check all the files in there and see which to keep and which to get rid of. So I'm thinking maybe we just move all of the stuff that is going to survive over into the `requests` directory, even though I agree with you that there will be a few files that are shared amongst multiple request types. wdyt?

Reply:
I made the `cache` directory in `responses` and added the sorted set fetch response there, mostly so we could easily separate them. I am alright moving it though, especially since sorted set fetch is one of the only responses that is reused.

Reply:
maybe we even move it back to responses dir after the old one is deleted, but i think it will be easier to track progress for now if we just plan on getting rid of that whole dir.